changelog.d/7892.bugfix

```diff
@@ -0,0 +1 @@
+Import ABC from collections.abc for Python 3 compatibility.
```

Comment:
```suggestion
Import `ABC` from `collections.abc` for Python 3.10 compatibility.
```

Reply:
Not sure if this is really a bugfix though. Not sure if it matters.

Reply:
I added it since changelog was mandatory in the contributing section.

> All changes, even minor ones, need a corresponding changelog / newsfragment entry. These are managed by Towncrier.

Reply:
Yes, I'm wondering if this should be a misc change instead of a bugfix. 😄

Reply:
I would agree on it being `misc` instead. Renaming the file to `changelog.d/7892.misc` is enough :)

Reply:
Thanks, renamed.